sqlitecluster/SQLiteNode.cpp

```diff
@@ -862,6 +864,16 @@ bool SQLiteNode::update() {
             }
         }
 
+        // If the majority of logged in full peers responds with abstain, then re-search.
+        bool majorityAbstained = abstainCount * 2 >= numLoggedInFullPeers;
+        if (allResponded && majorityAbstained) {
```

Comment:
We don't really need the `allResponded` check, if over half the cluster has abstained we're broken even if more nodes could respond.